docs/Getting-started.md

```diff
@@ -0,0 +1,52 @@
+# Start contributing to the Performance plugin
+This guide is intended to give details on how to contribute to the Performance plugin. From setting up the development environment to creating pull requests.
+
+## Prerequisites
+- Node.js
+- Docker
+- [Git](https://git-scm.com)
+
+## Setup the development environment
+
+### Clone the repository
+Start by cloning the repository into your local machine. To do this, run the following command in your terminal:
+`git clone https://github.com/WordPress/performance.git`
+
+### Install the local development dependencies
+To install the local development dependencies, run the following command in your terminal:
+`npm install`
+
+### Start the local WordPress environment
+The Performance plugin uses Docker and `[wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env) to start a local WordPress environment.
+
+First, make sure Docker is installed, up and running on your machine. Then, run `npm run wp-env start` to start the WordPress environment. The WordPress development site will be available at `http://localhost:8080`.
+
+### Stop the local WordPress environment
+The following command will stop the WordPress environment:
+`npm run wp-env stop`
+
+## Create a pull request
+A pull request must be created to submit changes to the Performance plugin. Pull requests should refer to an issue in the [repository issue tracker](https://github.com/WordPress/performance).
```

Comment:
Nit-pick, but maybe put the link for `https://github.com/WordPress/performance/issues` instead, since that's where the issues are?

Also I wonder if we should call this out a bit more explicitly, that it's recommended to open an issue to first discuss/review an idea _before_ opening a pull-request. It's not a strict requirement, but in most cases it makes sense, unless the PR is e.g. to fix a typo or something really minor.

I would go as far as saying maybe we should put another brief section about `Open an issue` above the `Create a pull request` section.

Reply:
Yeah, it makes sense to have an `Open an issue` section.